packages/client/webapp/src/utils/petrinet-validator.ts

```diff
@@ -0,0 +1,92 @@
+export interface Petrinet {
```

Comment:
Since "Petri net" is two words, this should be `PetriNet` and the other uses of the term in the other functions and variables in this file.
This seems nitpicky but it'll be much easier to be consistent later and avoid typos when they can't be automatically caught if we use the established spelling.